### PR TITLE
Extra data Menu for heatmap

### PR DESCRIPTION
**Pull request by maharit108**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -45,8 +44,8 @@
     ]
   },
   "devDependencies": {
-    "@testing-library/dom": "^7.28.1",
-    "@testing-library/user-event": "^12.5.0",
+    "@testing-library/dom": "^7.31.2",
+    "@testing-library/user-event": "^12.8.3",
     "concurrently": "^5.3.0",
     "json-server": "^0.16.3"
   }
```
Comment (MangeFre) [inline package.json:51]: Every change in package.json should be reverted. Even if it is a necessary change, let's save updates to all dependencies to for a different PR. So just unstage these changes from the PR 👍🏻

File: public/index.html [hunk 2]
```diff
@@ -2,15 +2,13 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
```
Comment (MangeFre) [inline public/index.html:4]: Are these changes necessary as well? Changing meta tags and such isn't bad or anything, but if we could keep PR's focused on only what they need to be, that'd make it easier to commit merges fast.

If these changes aren't necessary, it'd be best to not commit changes to index.html in this PR.

File: src/components/ExtraDataRightMenu/ExtraDataMenu.css [hunk 3]
```diff
@@ -0,0 +1,27 @@
+.extraData-wrapper{
+  margin: 0 16px 16px 16px;
+	background: white;
+  box-shadow: 0 1px 3px rgba(0, 0, 0, 0.2), 0 0.1px;
+	border-radius: 9px;
```
Comment (MangeFre) [inline src/components/ExtraDataRightMenu/ExtraDataMenu.css:5]: Clean up commented out code please 🙏🏻

File: src/components/Utility/filterFeatChart.js [hunk 4]
```diff
@@ -65,7 +64,19 @@ export const filterFeatChart = (data, selectedFeat) => {
         break
 
       case 'snap_data':
```
Comment (MangeFre) [inline src/components/Utility/filterFeatChart.js:66]: Remove console.logs as well please 🙏🏻

File: src/firebase/firebaseConfig.js [hunk 5]
```diff
@@ -1,10 +1,13 @@
-const firebaseConfig = {
```
Comment (MangeFre) [inline src/firebase/firebaseConfig.js:1]: Should this file really be commented out in main?

File: src/mapbox/CountyLayer.js [hunk 6]
```diff
@@ -10,6 +10,13 @@ const CountyLevel = () => {
      */
   const filters = useSelector(state => state.filters)
   const illinois_counties = useSelector(state => state.illinois_counties)
+
+  // const countyData = useSelector(state => state.county_data)
+  const selectedFeat = useSelector(state => state.selectedFeat)
+  const extraDataFeat = useSelector(state => state.extraDataMenuFeat)
+
```
Comment (MangeFre) [inline src/mapbox/CountyLayer.js:17]: Please remove this console.log as well 🙏🏻

**Review by MangeFre — CHANGES_REQUESTED** [review]
I think the code looks good, I just want you to confirm what truly needs to be part of this PR. 

If you go over my comments and either uncommit / revert some changes OR just explain why it needs to be part of this PR, then this looks fine to merge.

Might be some merge conflicts to resolve as well, but that's expected since this has been in the works for so long.

Overall it looks good, excited to see it all working in main and having some kind of final product 🙏🏻

File: package.json (hunk 1 above)
Comment (maharit108) [reply package.json]: Changes to package.json reverted.

File: src/components/Utility/filterFeatChart.js (hunk 4 above)
Comment (maharit108) [reply src/components/Utility/filterFeatChart.js:66]: console.logs removed. Have a look!!

**Review by MangeFre — APPROVED** [review]
This looks fine to me! Make sure you resolve any merge conflicts if there are any, and then go ahead and merge!! Awesome job!! 😎
